app/Nova/Actions/ActivateIntegration.php

```diff
@@ -59,12 +55,6 @@ public function fields(NovaRequest $request): array
                     'required',
                     'exists:organizations,id'
                 ),
-            Text::make('Coupon', 'coupon')
```

Comment:
Can't we put this behind a feature flag that for now we disable? Because we might want to reuse this in the future?